src/main/scala/net/bdew/ae2stuff/misc/AdvItemLocationStore.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) bdew, 2014 - 2015
+ * https://github.com/bdew/ae2stuff
+ *
+ * This mod is distributed under the terms of the Minecraft Mod Public
+ * License 1.0, or MMPL. Please check the contents of the license located in
+ * http://bdew.net/minecraft-mod-public-license/
+ */
+
+package net.bdew.ae2stuff.misc
+
+import net.bdew.lib.block.BlockRef
+import net.bdew.lib.nbt.NBT
+import net.minecraft.item.{Item, ItemStack}
+import net.minecraft.nbt.NBTTagCompound
+
+trait AdvItemLocationStore extends Item {
+
+  def addLocation(stack: ItemStack, loc: BlockRef, dimension: Int): Boolean = {
+    if (!stack.hasTagCompound) stack.setTagCompound(new NBTTagCompound)
+    val tag = stack.getTagCompound
+    if (tag.hasKey("dim") && tag.getInteger("dim") != dimension) {
+      false
+    }
+    val locList = tag.getTagList("loc", 10)
+    for (i <- 0 until locList.tagCount()) {
+      val tag = locList.getCompoundTagAt(i)
+      val pos = BlockRef.fromNBT(tag)
+      if (pos == loc) {
+        return false
+      }
+    }
+    locList.appendTag(NBT.from(loc.writeToNBT _))
+    tag.setTag("loc", locList)
+    tag.setInteger("dim", dimension)
+    print("Added location")
```

Comment:
This log should check for debug enabled feature

Reply:
Resolved by d0fa5cc

Reply:
This was also unintentionally left in - was removed